package.json

```diff
@@ -39,5 +45,10 @@
     "build:docs:api": "bash ./tools/update-api-reference.sh",
     "ci": "lerna run ci --ignore benchmark"
   },
-  "license": "MIT"
+  "lint-staged": {
+    "*.{js,ts,tsx,css}": [
+      "prettier --tab-width 4 --print-width 120 --write",
```

Comment:
> You shouldn't need to pass --tab-width 4 --print-width 120, it'll automatically find your config file.

Yes! I remove it after release next prettier. 

> Also you can do --ignore-path .eslintignore instead of having two files if you'd like.

Ah, I see.